lib/components/map/connected-transit-vehicle-overlay.js

```diff
@@ -0,0 +1,90 @@
+import TransitVehicleOverlay from '@opentripplanner/transit-vehicle-overlay'
+import coreUtils from '@opentripplanner/core-utils'
+import { Circle, CircledVehicle } from '@opentripplanner/transit-vehicle-overlay/lib/components/markers/ModeCircles'
+import { Tooltip } from 'react-leaflet'
+import { connect } from 'react-redux'
+import { FormattedMessage, injectIntl } from 'react-intl'
+
+const { formatDurationWithSeconds } = coreUtils.time
+
+const vehicleSymbols = [
+  {
+    minZoom: 0,
+    symbol: Circle
+  },
+  {
+    minZoom: 10,
+    symbol: CircledVehicle
+  }
+]
+
+function VehicleTooltip (props) {
+  const { direction, intl, permanent, vehicle } = props
+
+  let name = vehicle?.label
+  if (name !== null && name?.length <= 5) {
+    const mode = vehicle.routeType ? intl.formatMessage({
+      id: `common.otpTransitModes.${vehicle.routeType.toLowerCase()}`
+    }) : 'Line'
+    // Only render space if name is present
+    name = `${mode}${name && ' '}${name}`
```

Comment:
I thought about this and decided on not localizing it since I would expect my transit app to match what is shown on the vehicle. So in the case that the language would render this differently, wouldn't we still want it to match what is shown on the vehicle itself?

Reply:
Please add as a comment an example of the resulting text, and I don't see why `mode` would appear on the front window of the vehicle.

Reply:
Ah good call thank you! Forgot that that was rendered. Have localized

Reply:
in 140c81e8d652d4c488c4fcfaf3af10f2423ad954